src/FunctorOps.cpp

```diff
@@ -258,6 +258,24 @@ bool isInfixFunctorOp(const FunctorOp op) {
     return isInfixFunctorOp(xs.front().get().symbol);
 }
 
+FunctorOp getMinOp(const std::string& type) {
+    switch (type[0]) {
+        case 'f': return FunctorOp::FMIN;
+        case 'u': return FunctorOp::UMIN;
+        case 'i': return FunctorOp::MIN; assert("Unsupported Type");
```

Comment:
These asserts aren't used. If they're intended to be used, they would need to be the default behaviour.